packages/@cdklabs/cdk-cicd-wrapper/src/stacks/PipelineBlueprintBase.ts

```diff
@@ -64,43 +65,80 @@ export class PipelineBlueprintBase extends cdk.Stack {
     // [Backward compatibility] Allows to ignore stages which was env variable ACCOUNT_{STAGE} has discovered but value is set to - to ignore
     if (deploymentEnvironment.account == '-') return;
 
-    const phaseDefinition = this.resourceContext.get(GlobalResources.PHASE)!;
```

Comment:
Wont this remove the opportunity to override from outside?

Reply:
This line of code just moved closer to its usage. See preStageSteps function.